docs/source/release/v4.0.0/muon.rst

```diff
@@ -62,8 +63,9 @@ New
 
 Improvements
 ############
-- :ref:`MuonMaxent <algm-MuonMaxent>` has validation checks on its input.  
-
+- :ref:`MuonMaxent <algm-MuonMaxent>` has validation checks on its input.
+- :ref:`CalMuonDetectorPhases <algm-CalMuonDetectorPhases>` now sends a warning to the dialog window if it fails to fit a spectrum rather than throwing an exception and exiting the process of creating a phase table.
```

Comment:
`throwing an exception` is a very developer orientated phrase. Could we just have `now sends a warning to the dialog window if it fails to fit a  spectrum instead of exiting the process of creating a phase table.`